tfe/resource_tfe_policy_set_version.go

```diff
@@ -0,0 +1,99 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceTFEPolicySetVersion() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEPolicySetVersionCreate,
+		Read:   resourceTFEPolicySetVersionRead,
+		Delete: resourceTFEPolicySetVersionDelete,
+
+		Schema: map[string]*schema.Schema{
+			"policy_set_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policies_path_contents_checksum": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policies_path": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
Even though the `policies_path_contents_checksum` will change when anything in the `policies_path` changes, This `ForceNew` is needed, so as to avoid implementing an `Update` function. There isn't a notion of Update for PolicySetVersions.